### PR TITLE
Update docs with new axflow branding + fix logos

### PR DESCRIPTION
**Pull request by nichochar**
First pass at making the docs website follow our branding guidelines.
I removed the big logo since it was controversial (I still like it, but we can always re-introduce later).

I think that we can likely do a cleanup pass with some help from Rob, but otherwise this is good to go as a first pass. It also fixes the missing logos.

CleanShot 2023-09-20 at 14 43 31@2x
CleanShot 2023-09-20 at 14 43 25@2x

### CODE REVIEW

**Review by benjreinhart — COMMENTED** [review]
Can we remove the big logo on the right? It looks very odd to me

**Comment by nichochar** [comment]
I like it. Let's get a third opinion with Rob.

**Comment by benjreinhart** [comment]
Do we have an example of another pattern on a website where they have a giant logo on the right side of the page in a hero image spot? That's not necessarily a deal breaking reason, but I can't recall seeing something like that before and it stands out in an odd way

**Comment by nichochar** [comment]
It's the default design of vitepress, so every single one of theirs does this.
Examples: https://vitepress.dev/, https://adocs.vercel.app/

**Comment by benjreinhart** [comment]
The vitepress one looks good, but I think their logo is well suited for that. The other one you linked looks shitty imo. We could just leave it blank until we have something better to put there.

File: docs/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "docs",
   "private": true,
+  "type": "module",
```
Comment (nichochar) [inline docs/package.json:4]: This is important to fix a build issue:

Comment (nichochar) [inline docs/package.json:4]: This resolves a build issue and is an important change:
CleanShot 2023-09-20 at 14 47 54@2x

**Comment by nichochar** [comment]
[Vercel link](https://axflow-docs-git-rebrand-docs-axflow.vercel.app/)

**Review by benjreinhart — APPROVED** [review]

**Pull request merged**